posts/2022-04-12-java18-22004.adoc

```diff
@@ -0,0 +1,218 @@
+at java.base/java.text.MessageFormat.format(MessageFormat.java:885)
+at java.base/java.text.Format.format(Format.java:158)
+at java.base/java.text.MessageFormat.format(MessageFormat.java:860)
+at com.ibm.ws.logging.internal.impl.BaseTraceFormatter.formatMessage(BaseTraceFormatter.java:301)
+at com.ibm.ws.logging.internal.impl.BaseTraceFormatter.formatVerboseMessage(BaseTraceFormatter.java:373)
+at com.ibm.ws.logging.internal.impl.BaseTraceService.publishTraceLogRecord(BaseTraceService.java:1072)
+at com.ibm.ws.logging.internal.impl.BaseTraceService.publishLogRecord(BaseTraceService.java:1043)
+at com.ibm.ws.logging.internal.impl.BaseTraceService$2.publish(BaseTraceService.java:285)
+at java.logging/java.util.logging.Logger.log(Logger.java:979)
+at com.ibm.ws.logging.internal.WsLogger.log(WsLogger.java:138)
+at com.ibm.ws.logging.internal.WsLogger.logp(WsLogger.java:336)
+at com.ibm.ws.logging.internal.WsLogger.exiting(WsLogger.java:465)
+at com.ibm.ws.jsp.webcontainerext.JspDependent.(JspDependent.java:58)
+at com.ibm.ws.jsp.webcontainerext.AbstractJSPExtensionServletWrapper.loadClassInformation(AbstractJSPExtensionServletWrapper.java:233)
+at com.ibm.ws.jsp.webcontainerext.AbstractJSPExtensionServletWrapper.handleRequest(AbstractJSPExtensionServletWrapper.java:214)
+at com.ibm.ws.webcontainer.filter.WebAppFilterManager.invokeFilters(WebAppFilterManager.java:1227)
+at com.ibm.ws.webcontainer.webapp.WebApp.handleRequest(WebApp.java:5049)
+... 20 more
+----
++
+We've fixed this issue so that the `JSP` now returns to the client without a `NPE` even in file not found situations.
+
+[#guides]
+== New and updated guides since the previous release
+As Open Liberty features and functionality continue to grow, we continue to add link:https://openliberty.io/guides/?search=new&key=tag[new guides to openliberty.io] on those topics to make their adoption as easy as possible.  Since our last release, we've added the following two guides:
+
+* link:{url-prefix}/guides/openliberty-operator-intro.html[Deploying a microservice to Kubernetes by using Open Liberty Operator]
+** Explore how to deploy a microservice to Kubernetes by using Open Liberty Operator.
+* link:{url-prefix}/guides/openliberty-operator-openshift.html[Deploying a microservice to OpenShift 4 by using Open Liberty Operator]
+** Explore how to deploy a microservice to Red Hat OpenShift 4 by using Open Liberty Operator.
+
+Existing guides also receive updates in order to address any reported bugs/issues, keep their content current, and expand what their topic covers.  Since our last release, we've updated the majority of the guides to align with MicroProfile 5 and Jakarta EE 9.  Furthermore, our cloud hosted versions of the guides have also received various improvements.  For further information on these updates, refer to the link:{url-prefix}/blog/2022/04/12/guides-updated-mp5-jakartaee9.html[Learn MicroProfile 5.0 and Jakarta EE 9.1 from our newly enhanced cloud-hosted guides] blog post.
```

Comment:
```suggestion
We also update existing guides to address any reported bugs/issues, keep their content current, and expand what their topics cover.  Since our last release, we've updated the majority of the guides to align with MicroProfile 5 and Jakarta EE 9.  Furthermore, our cloud hosted versions of the guides have also received various improvements.  For further information on these updates, refer to the link:{url-prefix}/blog/2022/04/12/guides-updated-mp5-jakartaee9.html[Learn MicroProfile 5.0 and Jakarta EE 9.1 from our newly enhanced cloud-hosted guides] blog post.
```